README.md

```diff
@@ -5,48 +5,57 @@
 Display CPU frequency, usage, memory consumption, and battery percentage remaining within the VSCode status bar.
 
 ## Screenshots
-![An example of resmon running](images/example.png).
 
-![Disk space feature](images/ResmonDiskSpaceScreenshot.png).
+![An example of resmon running](images/example.png).
 
+![Disk space feature](images/disk_space_screenshot.png).
 
 ## Requirements
 
-Just the systeminformation node module.
+Just the system information node module.
```

Comment:
I had it be "systeminformation" since that's the name of the package.

I would be open to change it to "Just the `systeminformation` node module" so it's clear that it's a node package.

Reply:
You know, I'll just change that in a subsequent commit.

Reply:
Ohhh, my bad didn't realise!